src/service/cid-api.js

```diff
@@ -141,7 +143,7 @@ export class GoogleCidApi {
       setCookie(this.win_, cookieName, res['clientId'], this.expiresIn_(YEAR));
       return res['clientId'];
     } else {
-      this.persistToken_(TokenStatus.ERROR, DAY);
+      this.persistToken_(TokenStatus.NOT_FOUND, DAY);
```

Comment:
my mistake. I actually went ahead and changed it to 1 hour. Sorry about that. let's change it to 30 mins and look into further reducing if 1 hour doesn't seem adequate.